### PR TITLE
Create a README for the Concurrency TCK

### PR DESCRIPTION
**Pull request by KyleAure**
Document how to run the Concurrency TCK in Open Liberty either as a developer, or external verifier.

### CODE REVIEW

**Comment by KyleAure** [comment]
#build

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: Replace `This project is an implementation of the Jakarta Concurrency Technology Compatibility Kit (TCK) running on Open Liberty.`
with `This project runs the Jakarta Concurrency Technology Compatibility Kit (TCK) on Open Liberty.`

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: Replace `Concurrency-3.0` with Concurrency 3.0 or `concurrent-3.0`

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: "withe" should be "with the"
"version this keep this" should be "version. Keep this"

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: This section is important to be aware of almost from the beginning.  Can we use it as the "Overview"?  This will help it to make the most sense to the release approvers who are looking at this as documentation of how to run the official TCK against the Open Liberty beta.  That use case should be the most prominent focus, with the develop use case secondary.

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: Remove these lines.  Although they are true for now, we are writing this for when the beta does become available and will not submit the approval linking to this until then.

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: "setup" should be "set up"

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: "setup" should be "set up"

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/build.gradle
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/build.gradle]: Could you switch to 22.0.0.3-beta which is the first beta that will pass the Concurrency 3.0 TCK?
Also, let's leave out the jakartaee9 example given that the first EE spec level that we are automating the TCK for is EE 10

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/README.md]: Should include an example command to run the TCK with the default 3.0.0 version here.

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/bootstrap.properties
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/bootstrap.properties:12]: Was the trace accidentally removed?  Having it enabled shouldn't impact the ability to run the TCK.

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/configDropins/defaults/test.xml
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/configDropins/defaults/test.xml]: "Convience" should be "Convenience"

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/bootstrap.properties
Comment (njr-11) [reply dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/bootstrap.properties:12]: Never mind. I see where it was converted to server.xml later.

File: dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/tckRunner/tck/standalone.pom.xml
Comment (njr-11) [inline dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/tckRunner/tck/standalone.pom.xml:64]: In the meantime, use the copy of it that we uploaded to DHE, probably by making groupId also be variable? io.openliberty.jakarta.enterprise.concurrent
and version 3.0.0.20220120
The presence of the DHE version number is how we will figure out all of the places to update once everything is on Maven.

**Review by njr-11 — COMMENTED** [review]
Thanks for writing this up. It looks great. I have some minor suggestions for reordering some things to make running against the official TCK the most prominent.

**Review by njr-11 — APPROVED** [review]

**Comment by KyleAure** [comment]
#libbybot

**Comment by KyleAure** [comment]
Updates were to comments, text files, and an unused XML file. Going to merge based on the successful build from yesterday.

**Pull request merged**
